tests/cases/conformance/statements/for-inStatements/for-inStatements.ts

```diff
@@ -14,7 +14,7 @@ for (var x in fn()) { }
 for (var x in /[a-z]/) { }
 for (var x in new Date()) { }
 
-var c, d, e;
+var c: any, d: any, e: any;

```

Comment:
why add `any` here? Is it to preserve the intent of the tests &mdash; does control flow pick a union type for these variables now?